content/en/docs/userguide/try_waypoint.md

```diff
@@ -0,0 +1,136 @@
+  details-v1-5f4d584748-bz42z      1/1     Running   0          72s
+  productpage-v1-564d4686f-2rjqc   1/1     Running   0          72s
+  ratings-v1-686ccfb5d8-dnzkf      1/1     Running   0          72s
+  reviews-v1-86896b7648-fqm4z      1/1     Running   0          72s
+  reviews-v2-b7dcd98fb-nn42q       1/1     Running   0          72s
+  reviews-v3-5c5cc7b6d-q4r5h       1/1     Running   0          72s
+  sleep-9454cc476-86vgb            1/1     Running   0          62s
+  ```
+
+Test boofinfo works as expected
+
+  ```
+  [root@ ~]# kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+  ```
+
+Deploy waypoint:
+
+  Deploy a waypoint for service account `bookinfo-reviews`, so any traffic to service `reviews` will be mediated by that waypoint proxy
+
+  ```
+  [root@ ~]# istioctl x waypoint apply --service-account bookinfo-reviews
+  [root@ ~]# kubectl get pods
+  NAME                                               READY   STATUS    RESTARTS   AGE
+  bookinfo-reviews-istio-waypoint-5d544b6d54-v5tc9   1/1     Running   0          4s
+  details-v1-5f4d584748-bz42z                        1/1     Running   0          4m35s
+  productpage-v1-564d4686f-2rjqc                     1/1     Running   0          4m35s
+  ratings-v1-686ccfb5d8-dnzkf                        1/1     Running   0          4m35s
+  reviews-v1-86896b7648-fqm4z                        1/1     Running   0          4m35s
+  reviews-v2-b7dcd98fb-nn42q                         1/1     Running   0          4m35s
+  reviews-v3-5c5cc7b6d-q4r5h                         1/1     Running   0          4m35s
+  sleep-9454cc476-86vgb                              1/1     Running   0          4m25s
+  ```
+  
+  Replace the waypoint image with the Kmesh customized image. Based on istio-proxy, Kmesh adds an customized listener filter called [kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv) to connect L4 and L7.
+
+  ```
+  [root@ ~]# kubectl get gateways.gateway.networking.k8s.io
+  NAME               CLASS            ADDRESS         PROGRAMMED   AGE
+  bookinfo-reviews   istio-waypoint   10.96.207.125   True         8m36s
+  ```
+
+  Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0" to the `bookinfo-reviews` gateway, convert `{arch}` to the architecture of the host, current optional values are `x86` and `arm`. Then gateway pod will restart. Now kmesh is L7 enabled!
+
+### Apply weight-based routing
+
+Configure traffic routing to send 90% of requests to `reviews` v1 and 10% to `reviews` v2
+
+  ```
+  [root@ ~]# kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.21/samples/bookinfo/networking/virtual-service-reviews-90-10.yaml
+  [root@ ~]# kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.21/samples/bookinfo/networking/destination-rule-reviews.yaml
```

Comment:
Perhaps we can put these configurations in Kmesh's samples folder.
Or make it look like this:
```console
kubectl apply -f -<<EOF
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: reviews
spec:
  hosts:
    - reviews
  http:
  - route:
    - destination:
        host: reviews
        subset: v1
      weight: 90
    - destination:
        host: reviews
        subset: v2
      weight: 10
---
apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: reviews
spec:
  host: reviews
  trafficPolicy:
    loadBalancer:
      simple: RANDOM
  subsets:
  - name: v1
    labels:
      version: v1
  - name: v2
    labels:
      version: v2
  - name: v3
    labels:
      version: v3
EOF
```
More intuitively show users specific configurations. No need for the user to go to another project to view